lib/api/core/hotelClerk.js

```diff
@@ -574,14 +573,14 @@ class HotelClerk {
   }
 
   /**
-   * Delete room if no user has subscribed to it, and remove it also from the DSL
+   * Deletes a room if no user has subscribed to it, and removes it also from Koncorde
    *
    * @this HotelClerk
    * @param {string} roomId
    */
-  _removeRoomFromDsl (roomId) {
+  _removeRoomFromKoncorde (roomId) {
```

Comment:
nitpicking: I would use a more generic function name: `_removeRoomFromRealtime` instead of `..FromKoncorde`

Reply:
Comments and code now refer to the `real-time engine` instead of `Koncorde`